src/com/minseung/datastructure/LinkedListNode.java

```diff
@@ -7,6 +7,13 @@
     static class Node {
         int data;
         Node next = null;
+
+        Node() {
+        }
+
+        Node(int d) {
```

Comment:
parameter d does not give any idea to this method's user. Let's change this name to data